tests/modules/programs/gpg/override-defaults.nix

```diff
@@ -11,18 +11,17 @@ with lib;
         no-comments = false;
         s2k-cipher-algo = "AES128";
         throw-keyids = true;
-        trusted-key = [
-          "0xXXXXXXXXXXXXX"
-          "0xYYYYYYYYYYYYY"
-        ];
+        trusted-key = [ "0xXXXXXXXXXXXXX" "0xYYYYYYYYYYYYY" ];
       };
 
       homedir = "${config.home.homeDirectory}/bar/foopg";
     };
 
     nmt.script = ''
       assertFileExists home-files/bar/foopg/gpg.conf
-      assertFileContent home-files/bar/foopg/gpg.conf ${./override-defaults-expected.conf}
+      assertFileContent home-files/bar/foopg/gpg.conf ${
+        ./override-defaults-expected.conf
+      }
```

Comment:
Prime example why nixpkgs-fmt is superior: It does not enforce line lengths.

Reply:
I think this is very much a matter of taste. Personally, I don't mind this wrapping at all.